nilearn/input_data/tests/test_nifti_maps_masker.py

```diff
@@ -233,3 +233,33 @@ def test_nifti_maps_masker_2():
                                    masker.maps_img_.get_affine())
     assert_equal(fmri11_img_r.shape,
                  (masker.maps_img_.shape[:3] + (length,)))
+
+
+def test_nifti_maps_masker_overlap():
+    # Test resampling in NiftiMapsMasker
+    affine = np.eye(4)
+    shape = (5, 5, 5)
+    length = 10
+
+    fmri_img, _ = generate_random_img(shape, affine=affine,
+                                      length=length)
+    no_maps = np.zeros(shape + (2,))
```

Comment:
For the sake of clarity, what about expanding this `no_maps` into `non_overlapping_maps` and `o_maps` into `overlapping_maps`?